README.md

```diff
@@ -547,6 +547,7 @@ A curated list of awesome Java frameworks, libraries and software.
 * [Truth](https://github.com/google/truth) - Google's assertion and proposition framework.
 * [Unitils](http://www.unitils.org/) - Modular testing library for unit and integration testing.
 * [WireMock](http://wiremock.org/) - Stubbs and mocks web services.
+* [Greenmail](http://www.icegreen.com/greenmail/) - In-memory email server for testing integration testing email related operations. Supports SMTP, POP3 and IMAP including SSL 
```

Comment:
1. The library is written as "GreenMail"

2. Please rewrite the description to: 

In-memory email server for integration testing. Supports SMTP, POP3 and IMAP including SSL.